Description: fix missing Version to mean auto-Version as documented
 Having the Version member initialized to "1" if not present breaks the logic
 in _calcVersion.
Bug: https://rt.cpan.org/Public/Bug/Display.html?id=20297
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=849066

--- a/Barcode/QRcode.pm
+++ b/Barcode/QRcode.pm
@@ -37,7 +37,7 @@ sub init($$$){
     $oSelf->{text} = $sTxt;
     $oSelf->{Ecc} = $rhPrm->{Ecc} || ' ';
     $oSelf->{Ecc} =~ tr/LMHQ/M/c;    #Not /LMQH/ => M
-    $oSelf->{Version} = $rhPrm->{Version} || 1;
+    $oSelf->{Version} = $rhPrm->{Version} // 0;
     $oSelf->{ModuleSize} = $rhPrm->{ModuleSize} || 1;
     $oSelf->{ModuleSize} = int($oSelf->{ModuleSize});
 
